templates/cloudformation/iam_user_osdCcsAdmin.json

```diff
@@ -6,7 +6,8 @@
         "ManagedPolicyArns": [
           "arn:aws:iam::aws:policy/AdministratorAccess"
         ],
-        "UserName": "osdCcsAdmin"
+        "UserName": "osdCcsAdmin",
+        "Tags" : [{"Key": "MOA", "Value": "true"}]
```

Comment:
Since you're tagging the user, might as well check for the tag to verify that the `osdCcsAdmin` user was created by `moactl init`

Reply:
This is the plan, however right now the tagging is not working.
I filed an issue on AWS GO SDK repository: 
https://github.com/aws/aws-sdk-go/issues/3594

Reply:
@sulmer-redhat so this tag property isn't working?

Reply:
It is not working as of right now. should I remove it?